### PR TITLE
Fixed #715: Responsiveness

### LINKED ISSUE #715: Responsiveness
screen shot 2018-01-01 at 3 54 08 pm

Align them properly 
working on it

### PR DESCRIPTION
**Pull request by samyakbvs**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a Preview link and screenshot showning after and before the changes.
- [ ] Images are `240 x 240` [w x h].
- [x] Included a description of change below.
- [x] Squashed the commits.

# Changes done in this Pull Request
Responsiveness enhanced
- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**) https://samyakbvs.github.io/responsiveness-gci17.fossasia.org/
- Fixes #<Issue no.>715

### CODE REVIEW

**Comment by samyakbvs** [comment]
Before: 
screen shot 2018-01-01 at 5 45 27 pm
After: 
screen shot 2018-01-01 at 5 45 09 pm

**Comment by harshit98** [comment]
@samyakbvs The UI design is getting disturbed for laptop screen size:

After changes:

screen shot 2018-01-01 at 6 07 04 pm

Before changes:

screen shot 2018-01-01 at 6 07 00 pm

**Comment by samyakbvs** [comment]
Okay lemme work on that

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
A very bad PR title I must say !
And no need to change it to `absolute`
It will distort the layout in other dimensions !

Close this PR :+1:

**Comment by samyakbvs** [comment]
@0x48piraj If you will wait I will find solutions to all the problems

**Comment by 0x48piraj** [comment]
I don't think it's an issue @samyakbvs 
As seeing the Screenshot, I think you do not waited till the website loaded.

**Comment by samyakbvs** [comment]
@0x48piraj If you have a doubt that I didn't wait until the website loaded, you should go and check the website yourself

**Comment by 0x48piraj** [comment]
image

@samyakbvs See this!

**Comment by samyakbvs** [comment]
I checked it again @0x48piraj

**Comment by harshit98** [comment]
Don't argue with each other @0x48piraj @samyakbvs. If you want to argue, it should be technical. 

@samyakbvs Yes, take your time. Make sure the original laptop screen-size responsiveness is not disturbed. Also, change your commit name as well. It should be meaningful. How?
- When you'll squash your commits you can use `git commit --amend` to change the commit message.

@0x48piraj Thank you for reviewing the PR. 🎉

**Comment by samyakbvs** [comment]
Okay @harshit98

**Comment by samyakbvs** [comment]
Sorry by mistake clicked on close and comment

**Comment by 0x48piraj** [comment]
I was not arguing, just was trying to point out the possible error 
Anyways, @samyakbvs what's your resolutions ?
I'll test that in a simulator, and will notify @harshit98 what's the status. :+1:

**Comment by samyakbvs** [comment]
I have fixed it (almost)
There is a slight problem

**Comment by samyakbvs** [comment]
Actually now its solved completely

**Comment by 0x48piraj** [comment]
@samyakbvs Please don't throw practically useless comments :+1: 

Just make the changes, add a commit, provide a link. we're good to go !

**Comment by samyakbvs** [comment]
"Anyways, @samyakbvs what's your resolutions ?" 
I was just replying to you

**Comment by samyakbvs** [comment]
Done! @harshit98 and @0x48piraj  pls review

**Comment by 0x48piraj** [comment]
@samyakbvs Not good ! Yet, layouts are laggy

P.S. Converted the attr(s) to css and linked it :-1:

**Comment by samyakbvs** [comment]
I had to add more things that's why it was important to convert the attr(s) to css

**Comment by harshit98** [comment]
@samyakbvs The responsiveness of the website is still not in good condition:

- Small screen sizes:
screen shot 2018-01-01 at 7 13 16 pm

- Large mobile screen-size:

screen shot 2018-01-01 at 7 13 35 pm

- Tablet screen-size:

screen shot 2018-01-01 at 7 13 43 pm

**Comment by samyakbvs** [comment]
Laptop screen size:
screen shot 2018-01-01 at 7 26 18 pm
Tablet size: 
screen shot 2018-01-01 at 7 26 05 pm
Mobile size:
screen shot 2018-01-01 at 7 26 40 pm

Its working fine on my laptop and as you can see I am viewing the same link 
@harshit98 I think you forgot to refresh the page and even if you did pls do it one more time and then check :)

File: index.html [hunk 1]
```diff
@@ -324,7 +324,7 @@ <h2 class="number">Phimp.me</h2>
                 <p>
                     Do you have many different apps in your Android Phone for the purpose of clicking, editing and sharing images? Hey guess what, now you don't need all of those because Phimp.me is here, with all the features packaged into one. So go and clean your phone memory to save your space for other useful purposes.
                 </p>                
-                <div class="row" style="position:absolute; bottom:0px;left:30%">
+                <div class="row jig">
```
Comment (vaibhavsingh97) [inline index.html]: what do you mean by `jig`? can we have something relevant class name

File: css/theme-red.css [hunk 2]
```diff
@@ -3942,6 +3942,29 @@ video {
 .play-button.dark:before {
   border-color: transparent transparent transparent #292929;
 }
```
Comment (vaibhavsingh97) [inline css/theme-red.css:3944]: add one blank line after each class

File: css/theme-red.css [hunk 3]
```diff
@@ -3942,6 +3942,29 @@ video {
...
+
+@media only screen and (max-width: 990px) {
+    .jig {
+    position:absolute;
+    bottom:0px; 
+    top: 100%;
+    left: 20%;
+        }
+}
+@media only screen and (max-width: 770px) {
```
Comment (vaibhavsingh97) [inline css/theme-red.css:3959]: same

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]
Also, website is not responsive. Please fix all these issues
Screenshots:
image
image
There is enough space to have 2,3 colums
image
image
image
image
image
image
image
image

**Comment by samyakbvs** [comment]
I will do it @vaibhavsingh97 but I will need sometime

**Comment by samyakbvs** [comment]
Making a new PR

**Pull request closed without merging**